includes/Modules/AdSense.php

```diff
@@ -388,6 +400,17 @@ function( Google_Service_Adsense_Alert $alert ) {
 				}
 				$service = $this->get_service( 'adsense' );
 				return $service->accounts_adclients_urlchannels->listAccountsAdclientsUrlchannels( self::normalize_client_id( $data['accountID'], $data['clientID'] ) );
+			case 'GET:ad-blocking-recovery-tag':
+				if ( ! isset( $data['accountID'] ) ) {
+					return new WP_Error(
+						'missing_required_param',
+						/* translators: %s: Missing parameter name */
+						sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'accountID' ),
+						array( 'status' => 400 )
+					);
+				}
```

Comment:
We now have `Missing_Required_Param_Exception` which is a cleaner way to do this